packages/core/src/browser/style/notification.css

```diff
@@ -34,6 +34,10 @@
     text-align: center;
 }
 
+.notification-count-container-margin {
```

Comment:
I believe this file is strictly for notifications styling, shouldn't we put our styling under `view-container.css`?
The use of `notifications` for badge styling in `view-container` seems a little odd.